db/migrations/20160221001538_0.1.2_store-smtp-settings.sql

```diff
@@ -0,0 +1,41 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+ALTER TABLE campaigns ADD COLUMN smtp_id bigint;
+UPDATE campaigns 
+	SET smtp_id = (SELECT smtp.smtp_id FROM smtp) 
+	WHERE campaigns.id = (
+		SELECT smtp.smtp_id 
```

Comment:
From what I saw in testing the migrated ids are 1:1 as long as they were created through `/campaigns POST`